### DISCUSSION TITLE
Ability to easily see what is unmapped. (output as comment or trace)

### DISCUSSION THREAD

**Discussion by glenfinen** [riok/mapperly#975] [Ideas]
My apologies if this has been suggested or there is a way to do this already.  I think a nice feature would be an easy way to see what is unmapped.  Might be nice to output those fields as comments or a trace.  So it could be something like UnmappedMemberBehavior = None (default), Commented, Trace, Debug, Warning.  On a large object with mismatches, it would be an easy way to just look at the code that was generated and see what still needs a map.

**Comment by latonz** [comment]
If you enable [strict mappings](https://mapperly.riok.app/docs/configuration/mapper/#strict-property-mappings) each unmapped property will result in a RMG012 or RMG020. This should give you quickly the information whats unmapped. With the upcoming 3.3.0 release you can even configure [one-sided strict mappings](https://next.mapperly.riok.app/docs/configuration/mapper/#one-side-strict-property-mappings).
